### ISSUE TITLE
package.json: update dependencies

### ISSUE THREAD

**Issue by pkra** [mathjax/MathJax-node#159]

**Comment by pkra** [comment]
Renamed to be broader.

**Comment by pkra** [comment]
Hm... When I update to jsdom ^8 I get lots of errors in the tests -- but the tests all pass :frowning: 

Here are the errors I get (node 5.4.0)

test 1

```javascript
Error: ReferenceError: x is not defined
    at about:blank:1:1
    at Object.exports.runInContext (vm.js:44:17)
    at processJavaScript (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:71:10)
    at HTMLScriptElementImpl._eval (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:27:7)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:25:22
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:83:11)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:101:12
    at wrappedEnqueued (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:200:16)
Error: ReferenceError: x is not defined
    at about:blank:1:1
    at Object.exports.runInContext (vm.js:44:17)
    at processJavaScript (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:71:10)
    at HTMLScriptElementImpl._eval (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:27:7)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:25:22
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:83:11)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:101:12
Error: SyntaxError: Unexpected token ILLEGAL
    at Object.exports.runInContext (vm.js:43:16)
    at processJavaScript (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:71:10)
    at HTMLScriptElementImpl._eval (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:27:7)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:25:22
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:83:11)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:101:12
    at wrappedEnqueued (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:200:16)
    at ReadStream.<anonymous> (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:57:5)
    at emitNone (events.js:72:20)
```

test2

```
Error: ReferenceError: x is not defined
    at about:blank:1:1
    at Object.exports.runInContext (vm.js:44:17)
    at processJavaScript (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:71:10)
    at HTMLScriptElementImpl._eval (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:27:7)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:25:22
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:83:11)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:101:12
    at wrappedEnqueued (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:200:16)
Error: ReferenceError: x is not defined
    at about:blank:1:1
    at Object.exports.runInContext (vm.js:44:17)
    at processJavaScript (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:71:10)
    at HTMLScriptElementImpl._eval (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:27:7)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:25:22
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:83:11)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:101:12
```

test 3

```javascript
Error: ReferenceError: x is not defined
    at about:blank:1:1
    at Object.exports.runInContext (vm.js:44:17)
    at processJavaScript (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:71:10)
    at HTMLScriptElementImpl._eval (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:27:7)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:25:22
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:83:11)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:101:12
    at wrappedEnqueued (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:200:16)
Error: ReferenceError: x is not defined
    at about:blank:1:1
    at Object.exports.runInContext (vm.js:44:17)
    at processJavaScript (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:71:10)
    at HTMLScriptElementImpl._eval (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:27:7)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:25:22
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:83:11)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:101:12
ok 1 should be equal
Error: ReferenceError: x is not defined
    at about:blank:1:1
    at Object.exports.runInContext (vm.js:44:17)
    at processJavaScript (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:71:10)
    at HTMLScriptElementImpl._eval (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/HTMLScriptElement-impl.js:27:7)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:25:22
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:83:11)
    at Object.check (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:86:23)
    at /tmp/MathJax-node/node_modules/jsdom/lib/jsdom/living/nodes/Document-impl.js:101:12
    at wrappedEnqueued (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:200:16)
    at ReadStream.<anonymous> (/tmp/MathJax-node/node_modules/jsdom/lib/jsdom/browser/resource-loader.js:57:5)
```

**Comment by pkra** [comment]
This does not seem related to the patches for jsdom or updating to MathJax v2.6.

**Comment by pkra** [comment]
8 is pretty fresh so perhaps let's leave jsdom at `7.*` for now and we can check into this after 0.5 is out?

**Comment by dpvc** [comment]
It does seem to be in jsdom rather than MathJax.  I agree, leave at 7.* for now.

**Comment by pkra** [comment]
This was fixed by #160
